movie/models.py

```diff
@@ -11,10 +17,26 @@ class Movie(models.Model):
     def __unicode__(self):
         return "".join([self.name, " ", self.release])
 
+    # Add indexing method
+    def indexing(self):
+        obj = MovieIndex(
+            # meta = {'id : self.id'}, need not use that.
+            name = self.name,
+            release = self.release
+        )
+        obj.save(index='movie-index')
+        return obj.to_dict(include_meta=True)
```

Comment:
to_dict by default only serializes the data, not the metadata. If we want to see those as well, we have to do to_dict(include_meta=True). For now, i've not included metadata, but it's better to include it as to_dict just in case we add metadata in future.